pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -156,6 +156,21 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 		return err
 	}
 
+	discoveryClient, err := getDiscoveryClient()
+	if err != nil {
+		return err
+	}
+
+	apiGroupFilter, err := newApiGroupFilter(discoveryClient)
+	if err != nil {
+		return fmt.Errorf("unable to build api group filter: %v", err)
+	}
+
+	// Excluding tests with [apigroup:GROUP] labels for apigroups which are not
+	// served by a cluster. E.g. MicroShift is not serving most of the openshift.io
+	// apigroups. Other installations might be serving only a subset of the api groups.
+	tests = apiGroupFilter.filter(tests)
```

Comment:
In my ideal world, I'd get to see the test listed as skipped.  I think this is where that list gets produced https://github.com/openshift/origin/blob/master/pkg/test/ginkgo/junit.go#L30-L40 . Building on this, how much harder is it to produce that?